libvirt/network_def_test.go

```diff
@@ -50,8 +49,8 @@ func TestNetworkDefUnmarshall(t *testing.T) {
 			<ip address="192.168.122.1" netmask="255.255.255.0">
 				<dhcp>
 					<range start="192.168.122.100" end="192.168.122.254" />
-					<host mac="00:16:3e:77:e2:ed" name="foo.example.com" ip="192.168.122.10" />
-					<host mac="00:16:3e:3e:a9:1a" name="bar.example.com" ip="192.168.122.11" />
+					<host name="foo.example.com" ip="192.168.122.10" />
```

Comment:
I think I understand the removal above, as it fits the description of why we remove the bridge address, but I don't understand why we remove the matching in the DHCP host list.

Reply:
the mac here is only testing the unmarshalling mechanism as unit-test. 
We can leave it, yop.